repair/repair_hash.hh

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <absl/container/btree_set.h>
+#include <cstdint>
+#include <ostream>
+#include "schema.hh"
```

Comment:
It would make the header uncompilable on it's own with error "incomplete type schema used in type trait expression". The hasher has an instance of schema_ptr, which causes the problem. If it was just used in function params, fwd header would be enough.